eng/DownloadSharedSource.ps1

```diff
@@ -1,11 +1,17 @@
-$downloadPath = Resolve-Path (Join-Path $PSScriptRoot '..' 'src' 'assets' 'Azure.Core.Shared')
-$files = "ClientDiagnostics.cs", "ArrayBufferWriter.cs", "DiagnosticScope.cs"
-$baseUrl = "https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/core/Azure.Core/src/Shared/"
-
-foreach ($file in $files)
+function Invoke-DownloadSharedSource
 {
-    $text = Invoke-WebRequest -Uri "$baseUrl/$file";
-    $text.Content.Replace("#nullable enable", "#pragma warning disable CS8600, CS8604, CS8605").Trim() | Out-File "$downloadPath/$file"
-}
+    $currentProgressPreference = $ProgressPreference
+    $ProgressPreference = 'SilentlyContinue'
```

Comment:
Made the progress silent, as it has been a traditionally slow process in PowerShell.